chemprop/featurizers/molgraph/molecule.py

```diff
@@ -60,9 +60,9 @@ def __call__(
             )
 
         if n_atoms == 0:
-            V = np.zeros((1, self.atom_fdim))
+            V = np.zeros((1, self.atom_fdim)).astype(np.float32)
```

Comment:
```suggestion
            V = np.zeros((1, self.atom_fdim), dtype=np.single)
```
equivalently, `dtype="f4"`